benchmark/run.js

```diff
@@ -10,16 +10,26 @@ const cli = CLI(`usage: ./node run.js [options] [--] <category> ...
 
   --filter pattern          string to filter benchmark scripts
   --set    variable=value   set benchmark variable (can be repeated)
+  --format simple/csv       optional value that specifies the output format
 `, {
   arrayArgs: ['set']
 });
 const benchmarks = cli.benchmarks();
+const validFormats = ['csv', 'default'];
```

Comment:
Huh? I meant, in the usage text you added `--format` as `simple/csv`, but this array doesn’t seem to match that. Sorry if that’s been unclear (or I’m just misunderstanding this in general?)